package.json

```diff
@@ -268,5 +268,8 @@
     "whatwg-fetch": "^3.0.0"
   },
   "sideEffects": false,
+  "resolutions": {
+    "chokidar": "3.3.1"
+  },
```

Comment:
Is this related to the node 12/catalina crashes w/ webpack? https://github.com/webpack/watchpack/issues/130#issuecomment-606909820

I'm still on Mojave so I haven't ran into it (yet). Hopefully we can remove in the future if/when webpack 5 is released and storybook upgrades to use it instead of v4

```
❯ yarn why chokidar
yarn why v1.12.3
[1/4] 🤔  Why do we have the module "chokidar"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "chokidar@3.3.1" is incompatible with requested version "chokidar@^2.0.4"
warning Resolution field "chokidar@3.3.1" is incompatible with requested version "chokidar@^2.0.2"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "chokidar@3.3.1"
info Reasons this module exists
   - "webpack#watchpack" depends on it
   - Hoisted from "webpack#watchpack#chokidar"
   - Hoisted from "@storybook#react#react-dev-utils#fork-ts-checker-webpack-plugin#chokidar"
info Disk size without dependencies: "160KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "5.68MB"
info Number of shared dependencies: 49
✨  Done in 1.20s.
```

Reply:
yes exactly, @tay1orjones i thought it was low risk as it only impacts the development, not the runtime so squeezed that in to, apologies I forgot to mention it in the PR details